app/Policies/ProjectCommentPolicy.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App\Policies;
+use App\User;
+use App\ProjectComment;
+use Log;
+use Illuminate\Auth\Access\HandlesAuthorization;
+class ProjectCommentPolicy
```

Comment:
This is nice, but why the only policy? What's the benefit of doing this over the checking in Requests or on the controller like you're doing for the other methods?